src/components/sbb-toast/sbb-toast.tsx

```diff
@@ -0,0 +1,260 @@
+import {
+  Component,
+  ComponentInterface,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  JSX,
+  Method,
+  Prop,
+  State,
+} from '@stencil/core';
+import {
+  createNamedSlotState,
+  documentLanguage,
+  HandlerRepository,
+  languageChangeHandlerAspect,
+  namedSlotChangeHandlerAspect,
+  SbbOverlayState,
+} from '../../global/helpers';
+import { AriaPoliteness, ToastAriaRole, ToastPosition } from './sbb-toast.custom';
+import { isFirefox } from '../../global/helpers/platform';
+import { i18nCloseDialog } from '../../global/i18n';
+
+/**
+ * @slot unnamed - Use this to document a slot.
+ */
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-toast.scss',
+  tag: 'sbb-toast',
+})
+export class SbbToast implements ComponentInterface {
+  /** The length of time in milliseconds to wait before automatically dismissing the toast. */
+  @Prop() public timeout = 6000;
```

Comment:
I think it should be documented, that setting to "0" you can opt out of a timeout